spec/board_spec.rb

```diff
@@ -5,11 +5,18 @@
 
 describe TttBoard do
   it 'should initialize board with blank spaces' do
-    expect(subject.board_spaces).to eq({ '7': ' ', '8': ' ', '9': ' ', '4': ' ', '5': ' ', '6': ' ', '1': ' ',
-                                         '2': ' ', '3': ' ' })
+    expect(subject.board_spaces).to eq({ 0 => '00', 1 => '01', 2 => '02', 3 => '03', 4 => '04', 5 => '05', 6 => '06',
+                                         7 => '07', 8 => '08' })
   end
 
   it 'square 7 should be string with space " "' do
-    expect(subject.board_spaces[:'7']).to eq(' ')
+    expect(subject.board_spaces[7]).to eq('07')
+  end
+
+  context Board do
+    let(:board) { described_class.new(5) }
+    it 'should have a size of 9' do
+      expect(board.board_spaces.length).to be(25)
+    end
```

Comment:
This test is confusing - would like to see more descriptive `context`, something like 'initialized with row length 5'

and then
```
it 'should have a size of 25' do
```

Reply:
Yes, This should be 25, fixing.